src/components/Select/Select.style.ts

```diff
@@ -27,3 +28,14 @@ export const selectWrapper = ({
     cursor: ${!isSearchable && 'pointer'};
   }
 `;
+
+export const rightIconContainer = (
+  open: boolean,
+  isSearchable: boolean
+) => (): SerializedStyles => css`
+  display: flex;
+  gap: ${rem(25)};
```

Comment:
Can we discuss this with Design team to maybe use here a value from spacing?

Reply:
not sure gap (grid-gap) property has any effect on components with ```display: flex;```. Just gonna remove that